src/tasks/includes/utilities.js

```diff
@@ -39,6 +39,24 @@ const utilities = {
     }).thenReturn(results).all();
   },
 
+  validateThemeId: (themeId) => {
```

Comment:
This function returns a few different types. Ultimately we just want a true or false and then let the caller handle the error. I would go with just `if` statements and return a boolean each time. The try/catch and mixed types on return makes this a little confusing for future devs.